contentcuration/contentcuration/frontend/channelList/views/ChannelSet/ChannelSetModal.vue

```diff
@@ -226,9 +215,9 @@
     components: {
       CopyToken,
       ChannelSelectionList,
-      MessageDialog,
       ChannelItem,
       FullscreenModal,
+      KModal,
```

Comment:
Likewise, this line should be removed, we don't need to insert `KModal` as a child component.